Dockerfile

```diff
@@ -1,19 +1,16 @@
-FROM node:8.9.1
+FROM node:8.9.4-alpine
```

Comment:
Shouldn't we keep same node/engine major version for docker and application itself?

Reply:
Sorry missed it both are using same ver